src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -94,9 +95,15 @@ public static class ApplicationInsightsTelemetry
         // private const string _psCoreTelemetryKey = "ee4b2115-d347-47b0-adb6-b19c2c763808"; // Production
         private const string _psCoreTelemetryKey = "d26a5ef4-d608-452c-a6b8-a4a55935f70d"; // V7 Preview 3
 
+        // In the event there is a problem in creating the node identifier file, use the default identifier.
```

Comment:
Probably worth mentioning that this is typically within read only containers/env, unless it is unclear how many possible cases where there might be a problem creating node identifier.